### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by cheny19**
Merging Dev to Master branch.

Major change: Homopolymers simulation

### CODE REVIEW

File: src/simulator.py [hunk 1]
```diff
@@ -433,13 +435,76 @@ def read_profile(ref_g, ref_t, number, model_prefix, per, mode, strandness, exp,
...
+            hp_samples[ref_hp_end - ref_hp_start]["A/T"] = hp_samples[ref_hp_end - ref_hp_start]["A/T"][:-1]
+        else:
+            size = round(hp_samples[ref_hp_end - ref_hp_start]["C/G"][-1])
+            hp_samples[ref_hp_end - ref_hp_start]["C/G"] = hp_samples[ref_hp_end - ref_hp_start]["C/G"][:-1]
+
+        mutated_hp = base * int(size)
+        mutated_hp_with_mis = ""
+        for old_base in mutated_hp:
+            p = random.random()
+            if 0 < p <= 0.02215:
```
Comment (cheny19) [inline src/simulator.py]: @theottlo is this number basecaller dependent? If so, can we also put it into nd file?

Comment (theottlo) [reply src/simulator.py]: Sure I can put it there! That value (the mismatch rate in homopolymers) differs slightly between basecallers--0.02204, 0.02166, and 0.02215.

File: src/simulator.py [hunk 2]
```diff
@@ -1172,6 +1250,7 @@ def main():
                           type=float, default=None)
     parser_g.add_argument('--seed', help='Manually seeds the pseudo-random number generator', type=int, default=None)
     parser_g.add_argument('-k', '--KmerBias', help='Enable k-mer bias simulation', type=int, default=None)
+    parser_g.add_argument('-b', '--basecaller', help='Simulate k-mer bias from basecaller', default=None)
```
Comment (cheny19) [inline src/simulator.py]: list the three choices in the help info

File: src/simulator.py (hunk 1 above)
Comment (cheny19) [reply src/simulator.py]: It seems that I can't approve my own PR... so once you finish making the two changes, plz squash and merge, and go ahead with the benchmarking for the ms. Then I'll work on the release

**Pull request merged**